docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -325,16 +329,134 @@ When the proxy datasource is obtained via a JNDI lookup by the JDBC persistence
 
 The proxy datasource relies on the presence of the tenant-id and datastore-id information within the thread-local `FHIRRequestContext` information. In a simple configuration of the FHIR server that involves the use of the JDBC persistence layer implementation, the `FHIRRequestContext` information is obtained via request headers from the incoming REST API request. The tenant-id is obtained from the `X-FHIR-TENANT-ID` request header and the datastore-id is obtained from the `X-FHIR-DSID` request header<sup id="a3">[3](#f3)</sup>.
 
-#### 3.4.2.2 Datastore configuration examples
+#### 3.4.2.2 Standard JDBC Datasources
```

Comment:
You should check the table-of-contents.

Reply:
Yes, I plan to revise the TOC once we settle on the main changes.

Reply:
TOC is only 2 levels, so isn't changed.